grasa_event_locator/templates/provider.php

```diff
@@ -129,17 +130,39 @@
               {% if myEvent.isPending == 1 %}
                 <td><a href="{% url 'event_page' myEvent.id %}"><button type="button" class="btn btn-outline-info view-event" disabled>View</button></a></td>
                 <td><a href="{% url 'edit_page' myEvent.id %}"><button type="button" class="btn btn-outline-info editBtn" disabled>Edit</button></a></td>
-                <td><button type="button" class="btn btn-outline-danger" >Delete</button></td> 
+                <td><button type="button" class="btn btn-outline-danger" data-toggle="modal" data-target="#deleteModal" >Delete</button></td> 
               {% else %}
               
               <td><a href="{% url 'event_page' myEvent.id %}"><button type="button" class="btn btn-outline-info view-event">View</button></a></td>
               <td><a href="{% url 'edit_page' myEvent.id %}"><button type="button" class="btn btn-outline-info editBtn">Edit</button></a></td>
-              <td><button type="button" class="btn btn-outline-danger">Delete</button></td>
+              <td><button type="button" class="btn btn-outline-danger"  data-toggle="modal" data-target="#deleteModal">Delete</button></td>
               {% endif %}
+              <!--Delete Modal-->
+                <div class="modal fade" id="deleteModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel3" aria-hidden="true">
+                  <div class="modal-dialog" role="document">
+                    <div class="modal-content">
+                      <div class="modal-header">
+                        <h5 class="modal-title" id="exampleModalLabel3">Are You Sure?</h5>
+                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                          <span aria-hidden="true">&times;</span>
+                        </button>
+                      </div>
+                      <div class="modal-body">
+                       Are you sure you want to delete this Event? This cannot be undone.
+                      </div>
+                      <div class="modal-footer">
+                        <button type="button" class="btn btn-secondary" data-dismiss="modal">Never Mind</button>
+                        <button type="button" class="btn btn-danger" data-dismiss="modal" onclick="">Confirm Delete</button>
```

Comment:
This still needs to get hooked up, right?

Reply:
yes, its part of the list i sent you guys

Reply:
We should have that in a list of things to do somewhere.

Reply:
This is tracked in #79. :slightly_smiling_face: